File: src/list-add.erl
%
%	What Is This: programming samples
%	Author: Makoto Takeshita <takeshita.sample@gmail.com>
%	URL: http://simplesandsamples.com
%	Version: UNBORN
%	
%	Usage:
%	 1. git clone https://github.com/takeshitamakoto/sss.git
%	 2. change the directory name to easy-to-use name. (e.g. sss -> sample)
%	 3. open sss/src/filename when you need any help . 
%	

main(_)->
	Lis1 = [apple],
	Lis2 = lists:append(Lis1,[banana]),
	Lis3 = lists:append(Lis2,[1234]),
	Lis4 = lists:append(Lis3,[5678]),
	Lis5 = lists:append(Lis4,[candy]),
	io:format("~p~n",[Lis5]).

File: src/list-add.hs
{-
#	What Is This: programming samples
#	Author: Makoto Takeshita <takeshita.sample@gmail.com>
#	URL: http://simplesandsamples.com
#	Version: UNBORN
#	
#	Usage:
#	 1. git clone https://github.com/takeshitamakoto/sss.git
#	 2. change the directory name to easy-to-use name. (e.g. sss -> sample)
#	 3. open sss/src/filename when you need any help.
#	

-}

main = do let lis1 = []
              lis2 = reverse ("apple" : (reverse lis1))
              lis3 = reverse ("banana" : (reverse lis2))
              lis4 = reverse ("1234" : (reverse lis3))
              lis5 = reverse ("5678" : (reverse lis4))
              lis = reverse ("candy" : (reverse lis5))
          print lis

